models/vrp.rb

```diff
@@ -220,9 +220,18 @@ def self.check_consistency(hash)
       end
 
       # number of visits consistency
-      if !configuration[:schedule]
+      if configuration[:schedule]
+        if configuration[:schedule][:range_indices][:start] > 6
+          raise OptimizerWrapper::DiscordantProblemError.new('Schedule start index should be less or equal to 6')
+        end
```

Comment:
I don't think this is necessary and it might be limiting for some users and it will prevent our internal code from being generic. https://github.com/Mapotempo/optimizer-api/pull/158/files#r594430415

Reply:
I am not sure this is really limiting for our users. @braktar , @fab-girard can you give us a second opinion ?

Reply:
We can discuss it later : https://gitlab.com/mapotempo/optimizer-api/-/issues/701